sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -127,33 +127,35 @@ public abstract partial class NullableResponse<T>
         public abstract Azure.Response GetRawResponse();
         public override string ToString() { throw null; }
     }
-    public abstract partial class Operation
+    public partial class Operation
     {
         protected Operation() { }
-        public abstract bool HasCompleted { get; }
-        public abstract string Id { get; }
+        public Operation(Azure.Core.Pipeline.HttpPipeline pipeline, Azure.Core.RehydrationToken? rehydrationToken, Azure.Core.ClientOptions options = null) { }
```

Comment:
It would be preferable not to add a dependency of an end-user type (`Operation`) on a client-internal type (`HttpPipeline`).  Is there any way we could avoid this?  If you don't know offhand, we might be able to help you find design-alternatives if you could share the scenario that motivates the addition of this API.

Reply:
The actual implementation is hold in `NextLinkOperationImplementation`, so make `Operation` non-abstract, we need to implement the LRO rehydration based on `NextLinkOperationImplementation`.
To be able to construct `NextLinkOperationImplementation`, we need the user to input `HttpPipeline` and `RehydrationToken`, as construction [here](https://github.com/Azure/azure-sdk-for-net/pull/41088/files#diff-d98a5ed3ccb11d48b0a9d4261030c3b7d7487657ad6d54ec78c6d4984c2675e7R40).